modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/sql/GridSqlQueryParser.java

```diff
@@ -523,6 +523,15 @@
     /** Query parallelism value of cache configuration. */
     private static final String PARAM_PARALLELISM = "PARALLELISM";
 
+    /** */
+    private static final String PARAM_PK_INLINE_SIZE = "PK_INLINE_SIZE";
```

Comment:
I think we need add some docs for this new properties in "docs/_docs/sql-reference/ddl.adoc"